data/README.md

```diff
@@ -0,0 +1,151 @@
+---
+title: COVID-19 data mirrored in Galaxy
+---
+
+## Live Resources
+
+COVID-19 data is mirrored in the following Galaxy instances. No download needed.<br>
+
+| usegalaxy.eu | usegalaxy.org |
+|:--------:|:------------:|
+| <FlatShield label="usegalaxy" message="eu" href="https://bit.ly/usegalaxy-eu-covid19-data"/> | <FlatShield label="usegalaxy" message="org" href="https://usegalaxy.org/library/list#folders/Fe878daae442969ff"/> |
+| | |
+
+
+## COVID-19 resources available as data library
+
+The covid-19 folder of the shared data library contains a huge amount of SARS-CoV-2 data from public sources that we mirror here to streamline analyses on our server. We are also using this folder to deposit precomputed analysis results for further processing by you.
+
+Datasets that you are importing from the data library into your histories
+will be available almost instantaneously (no waiting time for downloads, no unexpected download interrupts)
+do not count towards your user quota
+help us avoid data duplication on our server
+
+The Export to History functionality of the shared data library offers powerful options for importing large data collections in structured ways.
+
+Currently, we host imaging, proteomics and next-generation sequencing (NGS) data on SARS-CoV-2.
+
+## NGS datasets in the data library
+
+For the NGS data, in particular, we are trying to offer streamlined access to our users to all SARS-CoV-2 public short reads data deposited in the NCBI SRA or at ENA.
+
+To keep this huge data collection manageable for us and you, we decided to organize the NGS data  further by sequencing technology used so inside a sequences folder you will find subfolders for Illumina-, Ion Torrent- and Nanopore-sequenced data. In addition, the sequences folder also contains:
+
+* a `current_metadata` dataset, with descriptions for all available NGS datasets,
+* the genbank dataset of the SARS-CoV-2 reference sequence NC_045512, which we suggest to use as the reference sequence in any of your analyses.
+
+The Illumina-sequenced data subfolder contains single-end and paired-end datasets, which can be distinguished by their names:
+the names of single-end datasets consist only of an SRR or ERR identifier followed by a file type suffix,
```

Comment:
We don't have a filetype suffix. I don't think that's useful since library datasets have a datatype that contains this information